### PR TITLE
Optimization for SEO, KW "Serverless Frameworks"

### PR DESCRIPTION
**Pull request by 97thFloor**
Updated with more instances of our focus keyword, as well as some internal links.

Primary KW: serverless frameworks
Secondary KWs: serverless, serverless deploy

@cnunciato @zchase

### CODE REVIEW

File: content/docs/intro/vs/serverless.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 title: Pulumi vs. Serverless Framework
-meta_desc: This page gives an overview of the major differences between Pulumi and
-           the Serverless Framework.
+meta_desc: Pulumi allows for a more holistic approach than the serverless framework, writing entire applications using an infrastructure-as-code approach.
```
Comment (ericrudder) [inline content/docs/intro/vs/serverless.md]:
```suggestion
meta_desc: Pulumi allows for a more holistic approach than the Serverless Framework, writing entire applications using an infrastructure-as-code approach.
```

Comment (ericrudder) [reply content/docs/intro/vs/serverless.md]: The casing matters here.

File: content/docs/intro/vs/serverless.md [hunk 2]
```diff
@@ -13,14 +12,14 @@ aliases: ["/docs/reference/vs/serverless/"]
 
 The Serverless Framework is a tool that makes programming AWS Lambda, Azure Functions, and Google Cloud Functions
 easier, by removing much of the boilerplate out of the native cloud providers' development experiences. It generates
-the YAML and configuration required for functions, and simplifies uploading code packs to the cloud. It relies on
+the YAML and configuration required for functions, and simplifies uploading code packs to the cloud. The Serverless Framework relies on
 the cloud provider's APIs or templating solutions for any provisioning or management beyond the functions themselves.
 
-Pulumi is more holistic in its focus. In Pulumi, you write entire applications -- some parts of which may be serverless
+Pulumi is more holistic than the Serverless Framework in its focus. In [Pulumi] (https://www.pulumi.com/blog/tag/pulumi/), you write entire applications -- some parts of which may be serverless
```
Comment (zchase) [inline content/docs/intro/vs/serverless.md]: @97thFloor What is the reason for linking to our blog instead of the home page? Just curious

Comment (97thFloor) [reply content/docs/intro/vs/serverless.md]: Good catch @zchase looks like that was a mistake on our end, let's change that to the homepage.

File: content/docs/intro/vs/serverless.md [hunk 3]
```diff
@@ -13,14 +12,14 @@ aliases: ["/docs/reference/vs/serverless/"]
 
 The Serverless Framework is a tool that makes programming AWS Lambda, Azure Functions, and Google Cloud Functions
 easier, by removing much of the boilerplate out of the native cloud providers' development experiences. It generates
-the YAML and configuration required for functions, and simplifies uploading code packs to the cloud. It relies on
+the YAML and configuration required for functions, and simplifies uploading code packs to the cloud. The Serverless Framework relies on
 the cloud provider's APIs or templating solutions for any provisioning or management beyond the functions themselves.
 
-Pulumi is more holistic in its focus. In Pulumi, you write entire applications -- some parts of which may be serverless
+Pulumi is more holistic than the Serverless Framework in its focus. In [Pulumi] (https://www.pulumi.com/), you write entire applications -- some parts of which may be serverless
```
Comment (zchase) [inline content/docs/intro/vs/serverless.md:18]:
```suggestion
Pulumi is more holistic than the Serverless Framework in its focus. In [Pulumi] ({{< relref "/" >}}), you write entire applications -- some parts of which may be serverless
```

File: content/docs/intro/vs/serverless.md [hunk 4]
```diff
@@ -30,3 +29,5 @@ the Serverless Framework](
 https://serverless.com/blog/serverless-application-for-long-running-process-fargate-lambda/) takes about 38 pages
 of explanation, including manual steps and AWS CloudFormation; [the after code using Pulumi](
 https://github.com/pulumi/examples/tree/master/cloud-js-thumbnailer) is only about 38 lines of code.
+
+Follow our blog for updates on [new features] (https://www.pulumi.com/blog/tag/new-features/), and if you have any questions about Pulumi, feel free to [contact us] (https://www.pulumi.com/contact/) any time.
```
Comment (zchase) [inline content/docs/intro/vs/serverless.md:33]:
```suggestion
Follow our blog for updates on [new features] ({{< relref "/blog/tag/new-features" >}}), and if you have any questions about Pulumi, feel free to [contact us] ({{< relref "/contact" >}}) any time.
```

**Comment by cnunciato** [comment]
Hey folks, we should either merge this or close it out. @zchase any qualms with my doing so?

File: content/docs/intro/vs/serverless.md [hunk 5]
```diff
@@ -30,3 +29,5 @@ the Serverless Framework](
 https://serverless.com/blog/serverless-application-for-long-running-process-fargate-lambda/) takes about 38 pages
 of explanation, including manual steps and AWS CloudFormation; [the after code using Pulumi](
 https://github.com/pulumi/examples/tree/master/cloud-js-thumbnailer) is only about 38 lines of code.
+
+Follow our blog for updates on [new features] ({{< relref "/blog/tag/new-features" >}}), and if you have any questions about Pulumi, feel free to [contact us] ({{< relref "/contact" >}}) any time.
```
Comment (cnunciato) [inline content/docs/intro/vs/serverless.md:33]: This paragraph seems bolted-on and doesn't seem to be adding much keyword-relevant info, so suggest simply removing it.
```suggestion
```

**Comment by cnunciato** [comment]
I'll get this merged.

**Comment by cnunciato** [comment]
Closing in favor of https://github.com/pulumi/docs/pull/3330. (Explanation there.)

**Pull request closed without merging**